.github/workflows/buildtest.yml

```diff
@@ -56,3 +56,56 @@ jobs:
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
           file: test/coverage/lcov.info
+
+  build-and-push-amd64-sriov-cni-on-pr:
+    name: Image Push AMD64
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to Docker
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and push sriov-cni
```

Comment:
this is going to accumulate allot of "junk" 

cant we build the image locally and use it ? as part of virtual-k8s-cluster step ?

Reply:
Good point. To use locally built images, we have to modify the scripts in the operator and push passed images to the k8s internal registry [run-e2e-conformance-virtual-cluster.sh#L226](https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/master/hack/run-e2e-conformance-virtual-cluster.sh#L226)

Another way to overcome this is to use a cleaner like [GHCR Delete Image](https://github.com/marketplace/actions/ghcr-delete-image), though I'm not sure how mature that project is

Reply:
can we re-use image published by https://github.com/k8snetworkplumbingwg/sriov-cni/blob/master/.github/workflows/image-push-master.yml?

Reply:
> can we re-use image published by https://github.com/k8snetworkplumbingwg/sriov-cni/blob/master/.github/workflows/image-push-master.yml?

`image-push-master.yml` publishes the image after PR is merged. With these changes, I'd like to run end-to-end tests against proposed changes. So, we need a way to build and publish (maybe in a temporary registry) an image different than the master one.